odoo/_monkeypatches/__init__.py

```diff
@@ -35,3 +35,5 @@ def patch_all():
     patch_werkzeug()
     from .zeep import patch_zeep
     patch_zeep()
+    from .datamatrix import patch_datamatrix_barcode
+    patch_datamatrix_barcode()
```

Comment:
This should be called `patch_reportlab`.

Nitpick: new line is missing at the end of file